compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/PipelineTranslator.java

```diff
@@ -107,7 +120,7 @@ private PipelineTranslator() {
   private static void boundedReadTranslator(final TranslationContext ctx,
                                             final PrimitiveTransformVertex transformVertex,
                                             final Read.Bounded<?> transform) {
-    final IRVertex vertex = new BeamBoundedSourceVertex<>(transform.getSource());
+    final IRVertex vertex = new BeamBoundedWindowSourceVertex<>(transform.getSource());
```

Comment:
`BeamBoundedSourceVertex` seems to be an unused class now.

How about 
(1) removing the current `BeamBoundedSourceVertex`, and 
(2) renaming `BeamBoundedWindowSourceVertex` to `BeamBoundedSourceVertex`?